webp_backport/src/main/java/de/marcreichelt/webp_backport/WebPBackport.java

```diff
@@ -38,17 +38,26 @@ static boolean isIsWebpSupportedNatively() {
     }
 
     /**
+     * Check if a given byte array is a WebP image
+     *
+     * @param encoded the byte array to check
+     * @return true if the byte array is recognized as a WebP image
+     */
+    public static boolean isWebP(byte[] encoded) {
+        return FileSignatureChecker.checkForWebP(encoded);
```

Comment:
As above, we simply could make FileSignatureChecker.checkForWebP public instead of doing a redirect here.